### PR TITLE
TDP-3632 Fix locale case

### PR DESCRIPTION
**Pull request by mrmeyers99**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

File: voice/bxml/verbs/speakSentence.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Attributes:
  * `level`: (optional) defines the strength of emphasis to be applied, accepted values are: `strong`, `moderate` or `reduced`
 
 #### `<lang>`
-Specifies the natural language of the content. If you use a voice with an `en_US` locale and an `es-MX` lang, it will sound like an English-speaking American attempting to speak Spanish.
+Specifies the natural language of the content. If you use a voice with an `en_US` locale, and a `es-MX` lang, it will sound like an American trying to speak Spanish.
```
Comment (vsmith-bw) [inline voice/bxml/verbs/speakSentence.md]: I think it should still be "an" 'es-MX'

**Review by rmello-daitan — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3632-case-fixup.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/verbs/speakSentence.md (hunk 1 above)
Comment (mrmeyers99) [reply voice/bxml/verbs/speakSentence.md]: oh, I didn't mean to change this paragraph.  I'll revert

**Review by vsmith-bw — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3632-case-fixup.s3-website-us-east-1.amazonaws.com/

**Pull request merged**